deploy/helm/jumpstarter/charts/jumpstarter-controller/templates/rbac/role.yaml

```diff
@@ -18,8 +18,6 @@ rules:
   - get
   - list
   - watch
-  - create
```

Comment:
wait this should have not been removed

Reply:
These are not removed, just the order changed.

Reply:
Now it's order yes, I force-pushed in the middle before you had another look:-) I found it because it was actually failing